plat/renesas/rcar/rcar_private.h

```diff
@@ -155,6 +155,9 @@ uint32_t rcar_get_spsr_for_bl32_entry(void);
 /* Gets the SPSR for BL33 entry */
 uint32_t rcar_get_spsr_for_bl33_entry(void);
 
+/* Declarations for rcar_scpi.c */
+uint32_t rcar_trigger_scpi(uint32_t x1, uint32_t x2, uint32_t x3, uint32_t x4);
```

Comment:
Why `uint32_t`? With 64-bit calling convention there can be 64 bit values.

Reply:
These arguments are not used inside function so far. Anyway, you are right and I will change to uint64_t.